src/cli_plugin/install/index.js

```diff
@@ -0,0 +1,47 @@
+import fromRoot from '../../utils/fromRoot';
+import install from './install';
+import Logger from '../lib/logger';
+import pkg from '../../utils/packageJson';
+import { parse, parseMilliseconds } from './settings';
+
+export default function pluginInstall(program) {
+  function processCommand(command, options) {
```

Comment:
Definitely not a big deal given that this is all part of a short lived cli command, but I don't think there's any reason to create this function every time `pluginInstall` is called.  It doesn't close over `program`, so it can probably live side by side.